_studio/mfx_lib/decode/mjpeg/src/mfx_mjpeg_dec_decode.cpp

```diff
@@ -680,6 +700,15 @@ mfxStatus VideoDECODEMJPEG::DecodeFrameCheck(mfxBitstream *bs, mfxFrameSurface1
         (*surface_out)->Info.CropX = 0;
         (*surface_out)->Info.CropY = 0;
 
+        mfxExtDecVideoProcessing * videoProcessing = (mfxExtDecVideoProcessing *)GetExtendedBuffer(m_vFirstPar.ExtParam, m_vFirstPar.NumExtParam, MFX_EXTBUFF_DEC_VIDEO_PROCESSING);
```

Comment:
Handle this Ext buffer in Query() method

Reply:
Done.